javascripts/AppConfig.js

```diff
@@ -10,17 +10,17 @@ app.config(function($routeProvider){
       templateUrl: 'partials/login.html',
       controller: 'LoginCtrl'
     })
-    .when("/favs", {
+    .when("/contacts/favs", {
       templateUrl: 'partials/contacts/favs.html',
```

Comment:
Make sure I can get to the right pages from the navbar, too :)

Reply:
what do you mean? like navbar/contacts/favs?

Reply:
Like, if I was a user, and I clicked on your navbar, it would bring me to the right route.

Reply:
but... according to part2 the navbar shouldn't even exist yet?

Reply:
Ah yes, but since you already have it mostly working, you may as well just fix what you have in this one. (You had it working before you updated the routes).